en/docs/deploy/secure-userstore-using-pbkdf2.md

```diff
@@ -0,0 +1,75 @@
+# PBKDF2 hashing
+We can use Password-Based Key Derivation Function 2 (PBKDF2) hashing method to securely store user passwords in user stores. This method reduces the risk of brute-force attacks due to insecure passwords.
+
+This guide walks you through the steps of configuring PBKDF2 as the hashing algorithm of a JDBC userstore.
+
+!!! note
+    Currently PBKDF2 supports only JDBC userstores in Asgardeo.
+
+## Prerequisite
+You should have a secondary JDBC userstore added.  [Create a JDBC user store](../configure-secondary-user-stores) if you don’t already have one.
+
+## Configure PBKDF2 hashing
+To configure PBKDF2 hashing on a JDBC user store: 
+
+1. On Identity Server console application (`https://<IS_HOST>:<PORT>/console`), go to  **Manage > Userstores**.
```

Comment:
Are we using the console now? Don't we need to explain this with the management console ?

Reply:
This is a mistake will update it. Thank you!